com.wso2telco.dep.hub.smsapi/smsapihub-synapse/src/main/synapse-config/sequences/sendSmsSeq.xml

```diff
@@ -20,6 +20,20 @@
     <foreach expression="//address" xmlns:ns="http://org.apache.synapse/xsd">
         <sequence>
             <property expression="//address/text()" name="MSISDN" scope="default" type="STRING"/>
+            <!-- if request is USER_ANONYMIZED then get actual msisdn -->
+            <filter regex="true" source="get-property('USER_ANONYMIZATION')">
+                <then>
+                    <property expression="//address/text()" name="MSISDN" scope="default" type="STRING"/>
+                    <class name="org.wso2telco.dep.nashornmediator.NashornMediator">
```

Comment:
NashornMediator needs to be replaced from UtilMediator since NashornMediator this will impact on performance

Reply:
Fixed in https://github.com/WSO2Telco/mediation-dep-hub-temp/pull/119/commits/fc4378f66ea8d6d2cff3b23dbecfaedf36c40afe